TimeSignal time_signal
CustomSignal custom_signal
string experiment_name
---
string response
